src/pages/blocks/centre.tsx

```diff
@@ -0,0 +1,327 @@
+    }
+
+    // Apply center type filter
+    if (centerType === 'regular') {
+      updatedCenters = updatedCenters?.filter((center) =>
+        ['Bhivapur', 'Kamptee', 'Katol'].includes(center)
+      );
+    } else if (centerType === 'remote') {
+      updatedCenters = updatedCenters?.filter((center) =>
+        ['Kuhi', 'Mauda', 'Savner'].includes(center)
+      );
+    }
+
+    // Apply selected centers filter
+    if (selectedCenters.length > 0) {
+      updatedCenters = updatedCenters?.filter((center) =>
+        selectedCenters.includes(center)
+      );
+    }
+
+    setFilteredCenters(updatedCenters);
+  };
+
+  useEffect(() => {
+    const storedDistrict = localStorage.getItem('selectedBlockDistrict') || '';
+    setDistrict(storedDistrict);
+    
+    const storedBlock = localStorage.getItem('selectedBlock') || '';
+    setBlocks(storedBlock);
+  }, []);
+
+  useEffect(() => {
+    applyFilters();
+  }, [sortOrder, centerType, selectedCenters]);
+
+
+  return (
+    <Box minHeight="100vh">
+      <Header />
+      <Box sx={{ display: 'flex', justifyContent: 'center' }}>
+        <Box
+          width={'100%'}
+          sx={{ backgroundColor: 'white', padding: '1.2rem' }}
+        >
+          <Typography
+            textAlign={'left'}
+            fontSize={'22px'}
+            color={theme?.palette?.text?.primary}
+          >
+            {blocks}
+          </Typography>
+          <Typography
+            textAlign={'left'}
+            fontSize={'11px'}
+            mb={'1rem'}
+            fontWeight={'500'}
+            color={theme?.palette?.text?.primary}
+          >
+            {districts}, Maharashtra
+          </Typography>
+          <Box display={'flex'} alignItems={'center'} mb={2}>
+          <TextField
+          value={searchInput}
+          onChange={handleSearchChange}
+          placeholder={t('COMMON.SEARCH')}
+          variant="outlined"
+          size="medium"
+          sx={{
+            p: 2,
+                justifyContent: 'center',
+                height: '48px',
+                flexGrow: 1,
+                mr: 1,
+                backgroundColor: '#EDEDED',
+                borderRadius: '40px',
+                '& .MuiOutlinedInput-notchedOutline': {
+                  border: 'none',
+                },
+                '& .MuiOutlinedInput-root': {
+                  boxShadow: 'none',
+                },
+          }}
+          InputProps={{
+            endAdornment: (
+              <InputAdornment position="start">
+                <Search />
+              </InputAdornment>
+            ),
+          }}
+        />
+          <Button
+              variant="outlined"
+              onClick={handleFilterModalOpen}
+              size="medium"
+              endIcon={<ArrowDropDown />}
+              sx={{
+                backgroundColor: 'white',
+                borderRadius: '7px',
+                border: '1px solid #EDEDED',
```

Comment:
@Aar-if colors to be moved to custom theme

Reply:
Okay